lesson_1/task_4.py

```diff
@@ -0,0 +1,17 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. 
+# Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, 
+# если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+# *Пример:*
+
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+
+
+s = int(input())
+katia = int((s/3)*2)
+petya = int((katia/2)/2)
+serega = int(petya)
```

Comment:
Использование функции `int` требуется, если использовать целочисленное деление.
```suggestion
katia = s // 3 * 2
petya = katia // 2 // 2
serega = petya
```